lib/docs/core/compat_tables.rb

```diff
@@ -0,0 +1,165 @@
+module CompatTables
+  require 'json'
+
+  TRANSLATEPATH = {
+    'element' => 'elements'
+  }
+
+  BROWSERS = {
+    'chrome' => 'Chrome',
+    'edge' => 'Edge',
+    'firefox' => 'Firefox',
+    'ie' => 'Internet Explorer',
+    'opera' => 'Opera',
+    'safari' => 'Safari',
+    'webview_android' => 'WebView Android',
+    'chrome_android' => 'Chrome Android',
+    'firefox_android' => 'Firefox for Android',
+    'opera_android' => 'Opera Android',
+    'safari_ios' => 'Safari on IOS',
+    'samsunginternet_android' => 'Samsung Internet'
+  }
+
+  GLOBALFILES = ['global_attributes']
+
+  def generateCompatibilityTable(doc, slug, documentation)
```

Comment:
I tried to make this somewhat abstract and not use any specific data from the scraper or the filters, this could be helpful if any other documentation needs this tables. However, this could be more abstract but my idea was not make an API of this, was just make html tables with the compatibility data.